code/modules/recycling/disposal.dm

```diff
@@ -486,7 +486,7 @@
 			attackby(dropping, user)
 		else if(istype(dropping, /obj/structure/closet/crate) && can_load_crates())
 			if(do_after(user,src,20))
-				if(dropping.locked_to || user.restrained() || !user.canmove)
+				if(dropping.locked_to || user.restrained() || !user.canmove || user.incapacitated() || !isturf(dropping.loc))
```

Comment:
you can drop the `user.restrained()` check since incapacitated() already does that. I'm not sure about the `canmove` check, I've never seen it